install/kubernetes/helm/istio/values.yaml

```diff
@@ -209,450 +290,3 @@ global:
   # Use the Mesh Control Protocol (MCP) for configuring Mixer and
   # Pilot. Requires galley (`--set galley.enabled=true`).
   useMCP: false
```

Comment:
now that everything has been abstracted, we can tidy up the globals :)

Reply:
indeed